ElectronApp/src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
It looks like this is a new dependency that wasn't added to package.json. Make sure the use the save-dev flag when adding a new dependency:

`npm install --save-dev @testing-library/user-event`